### PR TITLE
Revise JS SDK Doc Records section

### PR DESCRIPTION
**Pull request by TakLee96**
- Revise `content/js/record.md` for clarity
- Fix title in `menus/js-guide.json`
- Add style guide for JS docs in `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,3 +48,8 @@ Push to `production` branch to deploy to [http://docs.skygear.io](http://docs.sk
 │── package.json       # Dev dependencies and NPM scripts
 └── README.md          # Project overview
 ```
+
+## Style Guide for JS Docs
+- Four spaces per indent
```
Comment (rickmak) [inline README.md]: Two space, all other project *.js indent is two space across skygeario repos.

File: README.md [hunk 2]
```diff
@@ -48,3 +48,8 @@ Push to `production` branch to deploy to [http://docs.skygear.io](http://docs.sk
...
+- Use `console.error(error)` instead of `console.log(err)`
```
Comment (rickmak) [inline README.md]: => For logging error, please use `console.error(err)`. Don't use `console.log(err)` for unhanded error in example.

File: content/js/record.md [hunk 3]
```diff
@@ -1,117 +1,99 @@
...
 ``` javascript
 const Note = skygear.Record.extend('note');
 const Blog = skygear.Record.extend('blog');
+
+let note = new Note({ "content": "Hello World" });
+// { id: "...", recordType: "note", access: [Object object] }
 ```
 
-# Modify a record
+# (Create) Save a record
```
Comment (rickmak) [inline content/js/record.md]: I don't know if we want to have the brace `(Create)`. It look duplicate to me and title should be as short as possible?

File: content/js/record.md [hunk 4]
```diff
@@ -120,45 +102,68 @@ query.addDescending('popular');
...
+skygear.publicDB.save(new Note({
+    'id': '<your-note-id>',
+    'content': 'Hello New World!'
+})).then((record) => {
+    console.log(record);
 }, (error) => {
-  console.log(error);
+    console.error(error);
+});
+```
```
Comment (rickmak) [inline content/js/record.md]: Will this better? `new Note()` for modifying record look unconventional.

```
let note;
// Query the note

note['content'] = 'Hello New World!';
skygear.publicDB.save(note).then((record) => {
    console.log(record);
}, (error) => {
    console.error(error);
});
```

File: README.md (hunk 1 above)
Comment (TakLee96) [inline README.md]: Got it, will switch to two space and update the pull request

File: content/js/record.md (hunk 3 above)
Comment (TakLee96) [inline content/js/record.md]: Will make it more concise and update the pull request

File: content/js/record.md (hunk 4 above)
Comment (TakLee96) [inline content/js/record.md]: That does look better, will make a change

**Comment by rickmak** [comment]
You are mixing up too many change in this PR and I lost track. Let Keep each PR on modify one section only. For this PR, please try to append 4aa53b1e onto c1838417 and let me merge it.

**Comment by TakLee96** [comment]
I will submit a new pull request with only the first 4 commits and the response to your comment one

**Pull request closed without merging**